x-pack/plugins/cross_cluster_replication/server/routes/api/follower_index.js

```diff
@@ -274,6 +275,13 @@ export const registerFollowerIndexRoutes = (server) => {
           // Unfollow leader
           await callWithRequest('ccr.unfollowLeaderIndex', { id: _id });
 
+          // Try to re-open the index, store failures in a separate array to surface warnings in the UI
```

Comment:
Could this comment elaborate a bit on why we need to reopen them? Similar to how the description on this PR explains that they need to be open to be searched.